code-challenges.test.js

```diff
@@ -11,40 +11,259 @@
 // Run the file with the following command: $ yarn jest
 
 
-// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+// QUESTION_ONE) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
-// a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// const colors1 = (["purple", "blue", "green", "yellow", "pink"])
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+
+
+// const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 
-// b) Create the function that makes the test pass.
+// a) Create a test with an expect statement using the variable provided. 
+
+describe("colorsShuffle", () => {
+    it("removes the first item from the array and shuffles the remaining content", () => {
+        expect(colorsShuffle(colors1)).toEqual([ 'blue', 'green', 'pink', 'yellow' ])
+        expect(colorsShuffle2(colors2)).toEqual(['aquamarine', 'indigo', 'ochre', 'periwinkle', 'saffron' ])
+    })
+})
```

Comment:
Okay this is a good attempt and therefore acceptable. But your test already tells me we are on the wrong train tracks. The test should only be invoking a single function `colorsShuffle()` that takes in the parameters of the different colors each time it's invoked. 
So the goals are 
-> 1 function named colorsShuffle()
-> invoked 2 times
ex.
colorsShuffle(colors1)
colorsShuffle(colors1)
 ^ notice the invocation has the same name!